documentation/app/templates/components/list-item-template.hbs

```diff
@@ -0,0 +1,849 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<PageHeader
+  @title="List Item Template"
+  @figma="https://www.figma.com/file/SenC8kTQNUJKCvOvcL8WPu/Consul-Component-Library?type=design&node-id=722-120845&t=YhNf39KC0C4MRjwb-0"
+  @github="https://github.com/hashicorp/consul-ui-toolkit/blob/main/toolkit/src/components/cut/list-item/template/index.hbs"
+  @status="development"
+>
+  A List Item Template is used as a layout template to create a number of Consul
+  feature specific list items. It can also be used as a template for other items
+  that are able to make use of the template layout and its available
+  pre-determined element options.
+</PageHeader>
+<Hds::Tabs as |T|>
+  <T.Tab>Guidelines</T.Tab>
+  <T.Tab>Anatomy</T.Tab>
+  <T.Tab>Code</T.Tab>
+  <T.Panel>
+    <div class="doc-Page">
+      <div class="doc-Page-content">
+        <h2 id="usage">Usage</h2>
+
+        <h3 id="when-to-use">When to use</h3>
+        <ul>
+          <li>A List Item Template will most commonly be used within a
+            <Hds::Link::Inline @route="components.list-item">List Item
+              (container)</Hds::Link::Inline>. It can also be nested within
+            other components as a template.</li>
+        </ul>
+        <img
+          src="/images/list-item-template-usage.svg"
+          alt="List Item demo with List Item Template being used inside of it"
+        />
+
+        <h3 id="when-not-to-use">When not to use</h3>
+        <ul>
+          <li>When your content within is more than 3 lines tall, consider
+            creating your own template.</li>
+          <li>If you need to make use of styles or nested components that are
+            not already available within the template.</li>
+        </ul>
+
+        <h2 id="interactivity-and-actions">Interactivity & actions</h2>
+        <p>By default the List Item Template is static in nature. Interactivity
+          can be applied to elements within, but should adhere to the following
+          guidelines.</p>
+
+        <h3 id="tooltips">Tooltips</h3>
+        <p>Tooltips can be paired with the template for text, icon, or badge
+          elements that need to provide more context on hover. For example, when
+          the List Item Template is within a list item (container).</p>
+        <img
+          src="/images/list-item-template-tooltips.svg"
+          alt="Demonstration of tooltips on the List Item Template"
+        />
+
+        <h3 id="interactivity-specific-elements">Interactivity on a specific
+          element</h3>
+        <ul>
+          <li>
+            If paired with the List Item (container) the state of the container
+            should be set to “static”, meaning that clicking on the entire list
+            is non-interactive and only individual elements would be.
+          </li>
+          <li>
+            On hover of clickable data, the background color of the object
+            hovered should change to “Surface/interactive-hover”. If the action
```

Comment:
Might be convenient to link to the HDS color palette ref? https://helios.hashicorp.design/foundations/colors?tab=palette#surface-1